types/openfin/v29/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "dtslint/dt.json",
```

Comment:
For new packages, this file should only `"extends": "dtslint/dt.json"`, you can disable it in code like `// tslint disable-next-line max-line-length`